extensions-core/druid-pac4j/src/main/java/org/apache/druid/security/pac4j/Pac4jFilter.java

```diff
@@ -110,11 +110,11 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
               return profiles.iterator().next().getId();
             }
           },
-          NOOP_HTTP_ACTION_ADAPTER,
-          null, null, null, null);
```

Comment:
Changed the Authorizer from null to "none". In the older version, if it is null, it simply returns `authenticated and authorized -> grant access`. But in the 4.5.7 pac4j version, it uses CsrfAuthorizer as default, And because of this, I was getting 403 in a few tabs of druid UI. So, I have set it to "none".